packages/composite-checkout/src/types.ts

```diff
@@ -1,6 +1,12 @@
+import '@emotion/react';
 import { DataRegistry } from '@wordpress/data';
 import { ReactElement } from 'react';
-import { Theme } from './lib/theme';
+import { Theme as ThemeType } from './lib/theme';
+
+declare module '@emotion/react' {
```

Comment:
For anyone trying to understand what is going on here: https://emotion.sh/docs/emotion-11#theme-type

Reply:
yup. once you declare the theme on the module here, `props.theme` automatically gets the correct types in this kind of example:

```js
import styled from '@emotion/styled';

const Thing = styled.div`
  color: ${ ( props ) => props.theme.colors.highlight }; // `theme` type is inferred from our module declaration
`
```